index.html

```diff
@@ -34,6 +34,136 @@ <h2 class="connect">LET'S CONNECT</h2>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="backend">Back End Dev</span>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="fiftin">2015</span>
+                </div>
+                <article class="art">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article>
+                <ul class="frontend">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="seeproject" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="card1">
+            <img src="icons/Snapshoot Portfolio (3).png" alt="Snapshoot.png">
+            <div class="hero1">
+                <h2 class="Tonic">Multy Post <br>Stories</h2>
+                <div class="canopy">
+                    <span class="canopi">CANOPY</span>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="backend">Back End Dev</span>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="fiftin">2015</span>
+                </div>
+                <article class="art">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article>
+                <ul class="frontend">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="seeproject" type="button">See Project</button>
+            </div>
+        </div>
     </section>
+    <section class="about">
+        <div class="aboutmyself">
+            <h2 class="heading">About<br> Myself</h2>
+            <article class="main">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </article>
+            <h2 class="connect">LET'S CONNECT</h2>
+            <div>
+                <ul class="social">
+                    <li><a href="#"><img src="icons/Vector.png" alt="twitter_icon"></a></li>
+                    <li><a href="#"><img src="icons/Linkedin icon.png" alt="Linkedin_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (1).png" alt="Medium_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (2).png" alt="Github_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (3).png" alt="Chill_icon"></a></li>
+                </ul>
+            </div> 
+            <button type="button" class="seeproject">Get my resume</button>   
+        </div>
+        <div id="about">
```

Comment:
- [ ] I kindly suggest that you also use a two level nested list. i.e Make this `div` a `ul`, and then make nest the language, framework, and skills `ul's` in it's `li's`.

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```